### ISSUE TITLE
Multiple Words?

### ISSUE THREAD

**Issue by Omnipresent** [weinman/cnn_lstm_ctc_ocr#16]
This isn't an issue but a question. I've read the CRNN paper and have played around with PyTorch implementation of it: https://github.com/meijieru/crnn.pytorch

I've noticed that CRNN is able to detect single words but not multiple words. Example: 
![box_21](https://user-images.githubusercontent.com/115446/35488772-b70876b2-045b-11e8-8f63-6c8bb0df9b89.jpg)

My question is, would this project help with detection of multiple words?

**Comment by weinman** [comment]
To be clear, this software only does cropped text recognition. That is, it doesn't take an arbitrary image and identify the bounding boxes of the text (or words).

However, assuming you have something like the example above, this model could in theory be used to identify the separate words **if** you train with a literal space among the output character labels, and you generate training data with multiple words (containing the space character label). [This keras example](https://github.com/keras-team/keras/blob/master/examples/image_ocr.py) is an example that does so (with two words).

**Comment by Omnipresent** [comment]
I am using CTPN to prepare the cropped images from the bounding boxes it provides. It doesn't make bounding boxes on single words but on multiple words. Thanks for confirming that my training data would need a literal space. I will try playing around to see if there are other ways. Thanks
